File: Java/topflytechCodec/src/com/topflytech/codec/entities/BleCustomer2397SensorData.java
package com.topflytech.codec.entities;

/**
 * Created by admin on 2017/10/9.
 */
public class BleCustomer2397SensorData extends BleData{


    private String mac;
    private byte[] rawData;
    private Integer rssi;

    public byte[] getRawData() {
        return rawData;
    }

    public void setRawData(byte[] rawData) {
        this.rawData = rawData;
    }

    public Integer getRssi() {
        return rssi;
    }

    public void setRssi(Integer rssi) {
        this.rssi = rssi;
    }
}

File: TopflytechCodec/C#/TopflytechCodec/TopflytechCodec/Entities/BleCustomer2397SensorData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class BleCustomer2397SensorData : BleData
    {
        private String mac;
        private int rssi;

        public int Rssi
        {
            get { return rssi; }
            set { rssi = value; }
        }
        private byte[] rawData;

        public byte[] RawData
        {
            get { return rawData; }
            set { rawData = value; }
        } 
    }
}
